docs/quickstarts/registry.rst

```diff
@@ -0,0 +1,21 @@
+
+Flask-dance Demo Registry
```

Comment:
Flask-Dance has a capital D. Also, I think you can name this "Flask-Dance Examples" instead -- since you don't need to register your website, it's not really a "registry". 😄